src/test/java/pom/Homework22.java

```diff
@@ -0,0 +1,60 @@
+package pom;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework22 extends BaseTest {
+    @Test
+    public void renamePlaylist() {
+        String playlistName = "Daria46";
+//        LoginPage loginPage = new LoginPage(driver);
+//        HomePage homePage = new HomePage(driver) ;
+
+        loginPage.login();
+        homePage.doubleClickFirstPlayList();
+        homePage.enterPlaylistName(playlistName);
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A46-QA-Automation-Framework/assets/104761482/bcf19655-66b6-4643-b7e9-024e7ebe0b42)